src/index.js

```diff
@@ -0,0 +1,73 @@
+import './style.css';
+
+class TaskList {
+  constructor() {
+    this.size = 0;
+    this.ul = document.getElementById('task-list');
+    this.savedList = [{ description: 'Time to read', completed: false, index: 0 }, { description: 'Fix linters', completed: false, index: 1 }, { description: 'Do the homework', completed: false, index: 2 }];
```

Comment:
- [ ] On page load, the task array should be rendered dynamically to the DOM. The tasks should appear in order of the index values for each task.

Reply:
- [ ] Need to address this